pkgs/applications/networking/instant-messengers/chatterino2/default.nix

```diff
@@ -10,8 +10,15 @@ mkDerivation rec {
     sha256 = "0i2385hamhd9i7jdy906cfrd81cybw524j92l87c8pzrkxphignk";
     fetchSubmodules = true;
   };
-  nativeBuildInputs = [ qmake pkgconfig ];
+  nativeBuildInputs = [ qmake pkgconfig wrapQtAppsHook ];
```

Comment:
this already uses the custom deriver that add `wrapQtAppsHook`, so you don't need to add it here explicitly.

Reply:
I understand that the dependency is implied. I think it's clearer if the dependency is explicit, though.

Reply:
It's not implied, the deriver adds the dependency by design, and in 19.09 we had fix several 100+ expressions to use `mkDerivation` instead of `stdenv.mkDerivation` because people didn't read the docs.
It was even requested in the PR that initially was meant fix this https://github.com/NixOS/nixpkgs/pull/78296 https://github.com/NixOS/nixpkgs/compare/c2ce79000873d1936c07ec0261c200927c2bb40c..717c2207c2a0fe702ff678af230e583e05deb177

Reply:
> It's not implied, the deriver adds the dependency by design

That's what I meant when I said "the dependency is implied". The dependency is implied when using mkDerivation from pkgs/development/libraries/qt-5/mkDerivation.nix (instead of stdenv.mkDerivation).

On line 20, we call wrapQtApp in postFixup (on Darwin). wrapQtApp requires a dependency on wrapQtAppsHook. That's why I added the dependency here on line 13. If expressing the dependency here is a bug, let me know so I can remove it. My current understanding is that expressing the dependency here is merely *optional*.